tsc/meetings/2019-01-17.md

```diff
@@ -0,0 +1,103 @@
+Minutes from 5th OpenVDB TSC meeting, Jan. 17, 2019
+
+Attendees: John M., *Jeff* L., *Nick* A., *Ken* M., *Peter* C., *Dan* B., Andrew P.
+
+Additional Attendees: Bruce Chernia (Intel), Daniel Heckenberg (Animal Logic)
+
+Agenda:
+
+1) Confirmation of quorum
+2) Selection of secretary
+3) Update from TAC (Ken)
+4) Update on Multi-Res grid from Autodesk (Ken)
+5) Intel representative
+6) JIT for OpenVDB (Nic)
+7) Copyright notice (Dan)
+8) Write permissions (Dan)
+9) Release process (Dan)
+10) Cleanup and improve repository (Ken)
+11) Schedule next meeting
+
+1) A quorum was confirmed.
+
+2) Secretary - Nick Avramoussis.
+
+3) OpenColourIO is accepted within the probationary stage. There's been interest form 3rd party
+companies, including Nvidia, Intel and Autodesk.
+
+Security and CI:
+
+The issue of a security expert was discussed as something which should be addressed across all
+ASWF projects. Jeff mentioned that the guidelines are vague for this role and that in the context
+of VDB, this mainly relates to bug handling/invalid file reading. Ken mentioned the concern of
+arbitrary metadata in regards to attacks. Andrew advised on static code analysis tools as part
+of a CI build (lgtm, clang). Dan mentioned using clang tidy as a CI step which would be easier
+to solve on the CMake side to address CI best practices.
+
+The viability of Travis support was questioned. Previous limitations have been removed (Nov 2018)
+which now allow for building on Ubuntu 16.04 for Houdini 17. Dan has a pull request which
+optimises the Travis builds to help with timeouts, though it's agreed that it is not a sustainable
+solution. It was agreed that the main thing holding us back from a Jenkins transition was the
+CMake build.
+
+CMake:
+
+A fully supported CMake build through CI is becoming more of a priority. Current TSC members
+do not have extensive experience with CMake. Nick mentioned his experience and usage within
+DNEG production and will share example modification. It was agreed that other expertise would
+be ideal for deciding on a final structure. OpenColorIO (and others) have extensive CMake
```

Comment:
I think the suggestion here is not that the CMake system within OpenColorIO would become standardized, just that the relative level of experience for CMake within the OpenColorIO project is greater than the OpenVDB project so perhaps this is an area where OpenColorIO could lead the way on establishing and adhering to CMake conventions and OpenVDB follow. 

(see the number of contributors here for example - https://github.com/imageworks/OpenColorIO/blob/master/CMakeLists.txt)

Reply:
Yeah this is what I mean't I'll re-word to be clearer